tests/pre_merge/utils/callbacks/visualizer_callback/dummy_lightning_model.py

```diff
@@ -61,10 +61,10 @@ def test_step(self, batch, _):
         )
         return outputs
 
-    def validation_epoch_end(self, outputs):
+    def on_validation_epoch_end(self, outputs):
         return None
 
-    def test_epoch_end(self, outputs):
+    def on_test_epoch_end(self, outputs):
```

Comment:
this is the stuff we talked about this morning. Does `on_test_epoch_end` has `outputs` after v2+?